test/gurka/test_tagged_values.cc

```diff
@@ -137,7 +173,7 @@ TEST_F(TaggedNames, test_taking_tunnel) {
   }
 }
 
-TEST_F(TaggedNames, test_bypass_tunnel) {
+TEST_F(TaggedValues, test_bypass_tunnel) {
```

Comment:
so these are the places where i would have expected the tunnel and bridge tests. i saw you added more tests in your other pr. can you review these tunnle and bridge tests here and see if they already cover what you did in your new test and if they already cover it please remove the new one? for data style tests like this we sh oudl always use gurka, #3267

Reply:
So, yeah, here we have tests which actually test the same thing. Removing one I added in that PR.